packages/expo-updates/src/Updates.ts

```diff
@@ -32,13 +32,27 @@ export async function reloadAsync(): Promise<void> {
   if (!ExpoUpdates.reload) {
     throw new UnavailabilityError('Updates', 'reloadAsync');
   }
+  if (__DEV__) {
+    throw new CodedError(
+      'ERR_UPDATES_DISABLED',
+      'You cannot use the Updates module in development mode. To test manual updates, make a ' +
+        'release build with `yarn ios --configuration Release` or `yarn android --variant Release`.'
+    );
+  }
   await ExpoUpdates.reload();
 }
 
 export async function checkForUpdateAsync(): Promise<UpdateCheckResult> {
   if (!ExpoUpdates.checkForUpdateAsync) {
     throw new UnavailabilityError('Updates', 'checkForUpdateAsync');
   }
+  if (__DEV__) {
+    throw new CodedError(
+      'ERR_UPDATES_DISABLED',
+      'You cannot check for updates in development mode. To test manual updates, make a ' +
+        'release build with `yarn ios --configuration Release` or `yarn android --variant Release`.'
```

Comment:
probably don't want to couple this to yarn, `npm` is a bit more agnostic

Reply:
FWIW I don't use `yarn` so this build command doesn't help me. It's odd that this doesn't work in Development (and just crashes). I am seeing this on `expo-updates` ~0.1.2`. Will just test in production when I get a chance :)

Reply:
if you see the changes after the update you will notice we changed it to `npm`

so, `npm run ios --configuration Release` and the equivalent for android

Reply:
i would recommend upgrading to 0.2.4 of expo-updates, more info on it here https://blog.expo.io/over-the-air-updates-from-expo-are-now-even-easier-to-use-376e2213fabf :)